packages/protobuf-test/src/reflect/reflect.test.ts

```diff
@@ -403,8 +403,8 @@ describe("ReflectMessage", () => {
     });
     test("returns error setting float for int", () => {
       const f = desc.field.singularInt32Field;
-      const err = catchFieldError(() => r.set(f, 3.142));
-      expect(err?.message).toMatch(/^expected number \(int32\), got 3.142$/);
+      const err = catchFieldError(() => r.set(f, 3.14));
```

Comment:
Another curious change... The things you learn. I guess we could also use `Math.PI` 🤷

Reply:
This was a fun one :) In this case, we don't want to use Math.PI, but it's a good lint rule.